### ISSUE TITLE
TF Slim batch_norm does not expose beta_regularizer or gamma_regularizer

### ISSUE THREAD

**Issue by cancan101** [tensorflow/tensorflow#7367]
[`batch_normalization`](https://github.com/tensorflow/tensorflow/blob/b00fc538638f87ac45be9105057b9865f0f9418b/tensorflow/python/layers/normalization.py#L255-L256) takes in a `beta_regularizer` and `gamma_regularizer` but the [TF slim `batch_norm` layer does not expose this](https://github.com/tensorflow/tensorflow/blob/bd4a1c58d28734c77ecb17d719e72c45c6c33077/tensorflow/contrib/layers/python/layers/layers.py#L362-L380).

Compare this to other TF slim layers, such as `convolution` which do expose the regularizers: https://github.com/tensorflow/tensorflow/blob/bd4a1c58d28734c77ecb17d719e72c45c6c33077/tensorflow/contrib/layers/python/layers/layers.py#L789-L791

**Comment by girving** [comment]
@cancan101 Seems like a reasonable request.

@sguada I'll mark this contributions welcome, but let me know if there's a reason against it.

**Comment by cancan101** [comment]
Do you think it should have one regularizer param like for the `initializer` param on batch_norm? Or be consistent with how `convolution` has a param for each regularizer? (note: `convolution` also has a param for each `initializer` which is not consistent with `batch_norm`).

**Comment by sguada** [comment]
I think one param is a good idea.
